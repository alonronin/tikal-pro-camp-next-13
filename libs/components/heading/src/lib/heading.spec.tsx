import { render } from '@testing-library/react';

import Heading from './heading';

describe('ComponentsHeading', () => {
  it('should render successfully', () => {
    const { container } = render(<Heading>Hello</Heading>);
    expect(container).toMatchSnapshot();
  });
});
