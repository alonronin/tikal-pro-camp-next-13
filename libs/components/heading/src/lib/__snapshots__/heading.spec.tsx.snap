// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentsHeading should render successfully 1`] = `
<div>
  <h1
    class="font-bold text-2xl text-blue-500"
  >
    Hello
  </h1>
</div>
`;
